libclamav/matcher-ac.c

```diff
@@ -641,10 +641,35 @@ static void ac_free_special(struct cli_ac_patt *p)
     MPOOL_FREE(mempool, p->special_table);
 }
 
+/*Find all duplicate pointers and zero them so that we won't get double-free
+ * errors when we iterate over the cli_matcher to free all the trans
+ * pointers.*/
```

Comment:
Block-style comments formatted like this (with a `*` left border) usually end with the `*/` on a line by itself.